abi/mp/index.js

```diff
@@ -100,54 +100,60 @@ const mp200Schema = {
   // DeleteNetListingEvent: [UInt256], // ListId
   events: [
     {
-      name: "ListEvent",
-      args: [
-        {
-          type: "uint256",
-        },
-        {
-          type: "uint64",
-        },
-        {
-          type: "uint256",
-        },
+      "name": "ListEvent",
+      "args": [
         {
-          type: "address",
+          "type": "uint256",
+          "name": "listingId"
         },
         {
-          type: "(byte,byte[40])",
+          "type": "uint64",
+          "name": "contractId"
         },
-      ],
-    },
-    {
-      name: "BuyEvent",
-      args: [
         {
-          type: "uint64",
+          "type": "uint256",
+          "name": "tokenId"
         },
         {
-          type: "uint256",
+          "type": "address",
+          "name": "listAddr"
         },
         {
-          type: "address",
+          "type": "(byte,byte[40])",
+          "name": "listPrice"
         },
         {
-          type: "(byte,byte[40])",
+          "type": "uint64",
+          "name": "endTime"
         },
         {
-          type: "address",
-        },
-      ],
+          "type": "uint64",
+          "name": "royalty"
+        }
+      ]
     },
     {
-      name: "DeleteNetListingEvent",
-      args: [
+      "name": "BuyEvent",
+      "args": [
         {
-          name: "listId",
-          type: "uint256",
+          "type": "uint256",
+          "name": "listingId"
         },
-      ],
+        {
+          "type": "address",
+          "name": "buyer"
+        }
+      ]
     },
+    {
+      "name": "DeleteNetListingEvent",
```

Comment:
This would be `DeleteListingEvent` I believe

Reply:
OK, I've changed DeleteNetListingEvent to DeleteListingEvent in abi, impl, and type definition file